cpp/src/slice2swift/SwiftUtil.cpp

```diff
@@ -154,20 +154,23 @@ namespace
         }
     }
 
+    // TODO: fix this to emit double-ticks instead of single-ticks once we've fixed all the links.
```

Comment:
Swift links should be wrapped in double-quotes, and they use `/` as a separator, not `.`.

Previously we weren't emitting links at all, we just wrote the raw-text as is.
This PR doesn't make the full update to use double-ticks, because many of our links are actually broken. See: #3120.